test/integration/examples/apiserver_test.go

```diff
@@ -90,7 +90,7 @@ func TestAggregatedAPIServer(t *testing.T) {
 			"--kubeconfig", wardleToKASKubeConfigFile,
 		})
 		if err := wardleCmd.Execute(); err != nil {
-			t.Fatal(err)
+			t.Error(err)
```

Comment:
I'm going to assume the `Fatal` from the `waitForWardleRunning` call below covers this

Reply:
Oh, makes sense. But then again, why are we calling goroutine if we have a blocking call right after the `go func()` call?

Reply:
Maybe I can use a channel similar to the one below, to fail on error just in case `Execute()` also finds some other error not in `waitForWardleRunning`